resources/config.yml

```diff
@@ -2,6 +2,7 @@
 
 # Supported Languages:
 # English (English) = eng,
+# Romana (RO) = ro,
```

Comment:
Can you do like this:
English (English) = eng,
German (Deutsch) = deu,
Romanian (Romana) = rom